Tests/Provider/Mapquest/LocatorTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/**
+ * This file is part of the Meup GeoLocation Bundle.
+ *
+ * (c) 1001pharmacies <http://github.com/1001pharmacies/geolocation-bundle>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Meup\Bundle\GeoLocationBundle\Tests\Provider\Mapquest;
+
+use Meup\Bundle\GeoLocationBundle\Model\Address;
+use Meup\Bundle\GeoLocationBundle\Model\Coordinates;
+use Meup\Bundle\GeoLocationBundle\Factory\AddressFactory;
+use Meup\Bundle\GeoLocationBundle\Factory\CoordinatesFactory;
+use Meup\Bundle\GeoLocationBundle\Provider\Mapquest\Locator as MapquestLocator;
+use Meup\Bundle\GeoLocationBundle\Provider\Mapquest\Hydrator as MapquestHydrator;
+use Meup\Bundle\GeoLocationBundle\Tests\Provider\LocatorTestCase;
+
+/**
+ *
+ */
+class LocatorTest extends LocatorTestCase
+{
+    /**
+     * @param string $result_filename
+     *
+     * @return MapquestLocator
+     */
+    public function getLocator($result_filename)
+    {
+        $logger = $this
+            ->getMockBuilder('Psr\Log\LoggerInterface')
+            ->getMockForAbstractClass()
+        ;
+
+        return new MapquestLocator(
+            new MapquestHydrator(
+                new AddressFactory('Meup\Bundle\GeoLocationBundle\Model\Address'),
+                new CoordinatesFactory('Meup\Bundle\GeoLocationBundle\Model\Coordinates')
+            ),
+            $this->getClient($result_filename, __DIR__),
+            $logger,
+            'Fmjtd%7Cluu829ur2d%2Ca2%3Do5-9w1gla', // api_key
```

Comment:
It's a test ... maybe you should use `uniqid()` or generate a dummy api key. (or make reference to the api key of the test environment)
not your dev environment api key ... -_-'